.gitignore

```diff
@@ -6,3 +6,4 @@ pkg
 ext/bert/c/Makefile
 ext/bert/c/*.bundle
 ext/bert/c/*.o
+ext/bert/c/*.so
```

Comment:
why are we ignoring *.so files ? (What are they?)

Reply:
Output from building the C extension (similar to .bundle but on linux systems).